File: ios/MenCare/Core/DesignSystem/Component/Badges.swift
import SwiftUI

/// Small rounded status/category pill.
struct Pill: View {
    let text: String
    var container: Color = .stone
    var content: Color = .ink

    var body: some View {
        Text(text)
            .font(Typo.labelMedium)
            .foregroundStyle(content)
            .padding(.horizontal, 10)
            .padding(.vertical, 4)
            .background(Capsule().fill(container))
    }
}

File: android/core/designsystem/src/main/kotlin/com/devora/mencare/core/designsystem/component/Badges.kt
package com.devora.mencare.core.designsystem.component

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import com.devora.mencare.core.designsystem.theme.Ink
import com.devora.mencare.core.designsystem.theme.Stone

/** Circle with serif initials — stands in for people photos across the app. */
@Composable

/** Small rounded status/category pill. */
@Composable
fun Pill(
    text: String,
    modifier: Modifier = Modifier,
    container: Color = Stone,
    content: Color = Ink,
) {
    Box(
        modifier = modifier
            .clip(RoundedCornerShape(999.dp))
            .background(container)
            .padding(horizontal = 10.dp, vertical = 4.dp),
    ) {
        Text(text, style = MaterialTheme.typography.labelMedium, color = content)
    }
}
